### ISSUE TITLE
[RLlib] Ray RLLib Dependencies Version Information

### ISSUE THREAD

**Issue by chrisyeh96** [ray-project/ray#39405]
### Description

I am trying to figure out the proper versions for dependencies of Ray RLLib. I tried reading through the Ray documentation and the Ray source code, but I couldn't find the relevant information on my own, so I'm looking for some help.

Specifically, I'd like to know what versions of the following packages does Ray RLLib support?
* gymnasium
* pettingzoo
* pytorch
* tensorflow

Ideally, I'd like to see a table that looks like the following (I've put in hypothetical numbers for example):

| Ray RLLib | gymnasium | pettingzoo | pytorch | tensorflow |
|-----------|-----------|------------|---------|------------|
| 2.4.*     | 0.26.3    | 0.22.*     | 1.13.1  | 2.11       |
| 2.5.*     | ?         | ?          | ?       | ?          |
| 2.6.*     | ?         | ?          | ?       | ?          |

Also, I understand that there are several different "requirements.txt" files in the [python](https://github.com/ray-project/ray/tree/master/python) directory, but as someone who isn't part of the Ray RLLib team, it would be helpful to have a README to decipher the requirements.txt files.

**Comment by chrisyeh96** [comment]
Just wanted to follow up on this issue. Can the Ray Team please provide some information about the versions of `gymnasium`, `pettingzoo`, `pytorch`, and `tensorflow` supported by Ray RLLib?

**Comment by cszhu** [comment]
Hello! This P3 issue has seen no activity in at least a year. It will be closed in 2 weeks as part of ongoing cleanup efforts.

Please remove the `pending-cleanup` label if you believe this issue should remain open.

Thanks for contributing to Ray!
